File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    int i;
    int j;

    do
    {
        height = get_int("Please choose a number between 1 and 8, inclusive: \n");
    }
    while (height < 1 || height > 8);

    // Parent Loop
    for (i = 1; i <= height; i++)
    {
        // Left void
        for (j = height - i; j >= 1; j--)
        {
            printf(" ");
        }

        // Left triangle
        for (j = 1; j <= i; j++)
        {
            printf("#");
        }

        // Double space
        printf("  ");

        // Right triangle
        for (j = 1; j <= i; j++)
        {
            printf("#");
        }

        // Move to next row
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
def main():
    height = get_height()
    for i in range(1, height + 1):
        print(" "*(height - i), end="")
        for k in range(1, i + 1):
            print("#", end="")
            k += 1
        print("  ", end="")
        for l in range(1, i + 1):
            print("#", end="")
            l += 1
        print()
        i += 1


def get_height():
    while True:
        try:
            n = int(input("Please choose a height between number 1 and 8: "))
            if n > 0 and n < 9:
                return n
        except ValueError:
            print("Integer, please")
            

main()